src/main/java/org/broadinstitute/hellbender/tools/walkers/validation/basicshortmutpileup/BasicValidationResult.java

```diff
@@ -104,4 +118,112 @@ public String getFilters() {
     public long getNumAltSupportingReadsInNormal() {
         return numAltSupportingReadsInNormal;
     }
+
+    //----- The following two public static methods read and write files
+    public static void writeToFile(final List<BasicValidationResult> records, final File outputTable) {
```

Comment:
I think just `write` and `read` as method names are fine, but up to you.  Might also consider just `file` for the parameter names, or perhaps `outputFile` and `inputFile` (the distinction between the current `outputTable` and `tableFile` is not clear).

Reply:
done